docs/support-matrix.md

```diff
@@ -0,0 +1,50 @@
+# Compatible Storage Providers
+
+Ark supports a variety of storage providers for different backup and snapshot operations. As of version 0.6.0, a plugin system allows anyone to add compatibility for additional backup and volume storage platforms without modifying the Ark codebase.
+
+## Backup Storage Providers
+
+| Provider                  | Owner    | Contact                         |
+|---------------------------|----------|---------------------------------|
+| [AWS S3][2]               | Ark Team | [Slack][10], [GitHub Issue][11] |
+| [Azure Blob Storage][3]   | Ark Team |                                 |
+| [Google Cloud Storage][4] | Ark Team |                                 |
+
+## S3-Compatible Backup Storage Providers
+
+Ark implements [Amazon's Go SDK][12] to connect to the S3 API. Some third-party storage providers also support the S3 API, and users have reported the following providers work with Ark:
+
+_Note that these providers are not regularly tested by the Ark team._
+
+ * [IBM Cloud][5]
+ * [Minio][9]
+ * Ceph RADIOS v12.2.7
+
+## Volume Snapshot Providers
+| Provider                         | Owner           | Contact                         |
+|----------------------------------|-----------------|---------------------------------|
+| [AWS EBS][2]                     | Ark Team        | [Slack][10], [GitHub Issue][11] |
```

Comment:
I'd put the contact stuff in each Ark Team row, or find a way to merge the cells

Reply:
Looks like GH markdown doesn't support merging cells, and I don't want to mix HTML tables in, so I'll just duplicate it.